fixtures/tests/_fixtures/test_logger.py

```diff
@@ -140,6 +140,13 @@ def test_logging_output_included_in_details(self):
         except:
             pass
 
+    def test_exceptionraised(self):
+        fixture = self.useFixture(FakeLogger())
+        self.assertRaises(TypeError,
+                          logging.info,
+                          "some message",
+                          "wrongarg")
```

Comment:
This works but I'd be tempted to write it like so:

 ```
with FakeLogger():
     with testtools.ExpectedException(TypeError):
         logging.info("Some message", "wrongarg")
```
As it will be shorter and easier to read.